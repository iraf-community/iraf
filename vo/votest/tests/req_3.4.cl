#
#  Cache Registry query results.

# Set the test description string.
votest.descr = "Cache Registry query results."

print ("------------------------------------------------------------------")
print ("Req  3.4:  Users shall have the option to cache query search")
print ("	   results to avoid redundant service calls.")
print ("------------------------------------------------------------------")


# Execute the test commands.
print ("This is demonstrated by using the REGDB task to retrieve record")
print ("information from the local registry cache.\n\n")

regdb ("url", alias="dss", verb+)
regdb ("url", alias="nvss", verb+)

