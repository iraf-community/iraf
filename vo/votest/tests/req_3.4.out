------------------------------------------------------------------
Req  3.4:  Users shall have the option to cache query search
	   results to avoid redundant service calls.
------------------------------------------------------------------
This is demonstrated by using the REGDB task to retrieve record
information from the local registry cache.


http://skyview.gsfc.nasa.gov/cgi-bin/vo/sia.pl?survey=dss2&
http://skyview.gsfc.nasa.gov/cgi-bin/vo/sia.pl?survey=nvss&
http://heasarc.gsfc.nasa.gov/cgi-bin/vo/cone/coneGet.pl?table=nvss&
